app/controllers/application_records_controller.rb

```diff
@@ -75,13 +75,11 @@ def toggle_saved_for_later
       @record.move_to_dashboard
       flash[:message] = 'Application moved back to dashboard.'
     else
-      date = if params[:date_for_later].present?
-               Date.strptime(params[:date_for_later], '%m/%d/%Y')
-             end
-      mail = true if params[:mail_to_applicant] == '1'
-      @record.save_for_later(date: date,
+      parameters = save_for_later_params
+      @record.save_for_later(date: parameters[:date],
                              note: params[:note_for_later],
```

Comment:
That method interpolates params. No interpolation needed for the note for later.